plugins/drivers/proto/driver.proto

```diff
@@ -280,6 +284,36 @@ message ExecTaskResponse {
     ExitResult result = 3;
 }
 
+message ExecTaskStreamingIOOperation {
+    bytes data = 1;
+    bool close = 2;
+}
+
+message ExecTaskStreamingRequest {
+    message Setup {
+        string task_id = 1;
+        repeated string command = 2;
```

Comment:
Why is this repeated?

Reply:
command is a `[]string` , so gets mapped to `repeated string` in protobuf.